Pod/Classes/DeviceSize/DeviceSize.swift

```diff
@@ -74,13 +74,23 @@ public extension DeviceSize {
         }
     }
 
-    var hashValue: Int {
-        let a: Int = Int(dimensions.width)
-        let b: Int = Int(dimensions.height)
 
-        // Using Cantor's pairing function:
-        return (a + b) * (a + b + 1) / 2 + b
-    }
+    #if swift(>=5.0)
```

Comment:
@ZevEisenberg Shouldn't this be `#if swift(>=4.2)`?

Reply:
It’d be great if you could test it. I may have gotten it wrong, but I remember trying it out and landing on this for some reason. Should have added a comment.

Reply:
Will take a look! Might look into adding multiple Xcode build environments to the Circle YML tomorrow so we can test against Swift 4.2 and Swift 5.0